util/chplenv/chpl_bin_subdir.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import optparse
+import os
+from sys import stderr, stdout
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import chpl_home_utils
+import chpl_platform, chpl_compiler, chpl_machine, chpl_arch
+
+from utils import memoize
+
+@memoize
+def get(location):
```

Comment:
I find location to be a confusing name. Most chplenv scripts just call this "flag", which is relatively meaningless, but I don't find that confusing. location just doesn't seem like an accurate description to me.

Reply:
I copied it from chpl_arch. I'm happy to change it to flag in both places.